### ISSUE TITLE
[Serious] Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#64]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87559 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Account / Change Password |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/users/my_account> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 4:31 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

The contrast ratio between text and its background is not at least 4.5:1.
When a user changes passwords on the Change Password page the Your password has been changed message is dark gray on blue background with a low contrast.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

How To Fix: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Source Code

```
<div class="row flash">
        Your password has been changed.
      </div>
```

## Screenshots

![02_black blue contrast](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87559/02_black_blue_contrast.png)

**Comment by mjiori** [comment]
Ablr verified this fix in demo: Marked as "Fixed, RJ, 8/17/26"

The fix is now in prod. Closing.
